system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# description: ManageIQ appliance console launcher
+#
+
+if [ -d /var/www/miq/vmdb ] ; then
+  cd /var/www/miq/vmdb
+else
+  cd $(dirname $0)/../../../vmdb
```

Comment:
I agree, but will we run `system/LINK/bin/fix_auth SOME_OPTIONS`?

I guess I don't understand how that's easier than
`bundle exec ruby tools/fix_auth.rb SOME_OPTIONS`